tests/integration_tests/conftest.py

```diff
@@ -28,7 +28,7 @@ def lobby_server(request, loop, player_service, game_service, geoip_service):
     server.NatPacketServer.instance = NatPacketServer(addresses=server.config.LOBBY_NAT_ADDRESSES, loop=loop)
     loop.run_until_complete(server.NatPacketServer.instance.listen())
     ctx = run_lobby_server(
-        address=('127.0.0.1', None),
+        address=('127.0.0.1', 8002),
```

Comment:
Any particular reason for this?

Reply:
Only because the logging statement that tried to print the port number was failing. I can probably undo this, since i don’t think that was the reason why the test was failing.

Reply:
All right.

If it works without I think it's best to keep the port randomly assigned: It makes sure to use an unbound port this way, and we avoid faux errors where someone has another service running on it.